src/lib/builders/select/types.ts

```diff
@@ -150,8 +150,33 @@ export type SelectOptionProps<Value = unknown> = SelectOption<Value> & {
 	disabled?: boolean;
 };
 
-export type Select = BuilderReturn<typeof createSelect>;
-export type SelectElements = Select['elements'];
-export type SelectOptions = Select['options'];
-export type SelectStates = Select['states'];
-export type SelectHelpers = Select['helpers'];
+export type Select<
+	Value = unknown,
+	Multiple extends boolean = false,
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Any is not being used here, is it?

Reply:
Don't think so, honestly I just copied it over from the example you gave lol. I don't see any reason why it's needed though